Aztec/Classes/Public/Formatters/AttributeFormatter.swift

```diff
@@ -0,0 +1,148 @@
+import UIKit
+
+/// A type that provides support for toggling compound attributes in an
+/// attributed string.
+///
+/// When you want to represent an attribute that does not have a 1-1
+/// correspondence with a standard attribute, it is useful to have a virtual
+/// attribute. Toggling this attribute would also toggle the attributes for its
+/// defined style.
+///
+protocol AttributeFormatter {
```

Comment:
Well, we agree.  My point was more about it being fine to tackle those in an upcoming PR.  But your call in any case.